108_Convert_Sorted_Array_to_Binary_Search_Tree.py

```diff
@@ -0,0 +1,27 @@
+# Time: O(logn)
+# Space: O(n)
+from typing import List
+
+
+class TreeNode:
+    
+    def __init__(self, val=0, left=None, right=None):
+        self.val = val
+        self.left = left
+        self.right = right
+
+
+class Solution:
+    
+    def sortedArrayToBST(self, nums: List[int]) -> Optional[TreeNode]:
+        def createNode(start, end, nums):
```

Comment:
Add type annotation.
Also, functions should be named according to the snake case naming convention.